demos/jans-tarp/src/options/authFlowInputs.tsx

```diff
@@ -181,6 +181,7 @@ export default function AuthFlowInputs({ isOpen, handleDialog, client }) {
           console.log("userDetails: " + JSON.stringify(userInfoResponse.data));
```

Comment:
Please remove console.logs.

Reply:
Please let it be as it is , it is for displaying user-info on console.